packages/marketplace/contracts/TransferManager.sol

```diff
@@ -354,6 +445,19 @@ abstract contract TransferManager is Initializable, ITransferManager {
     /// @param from Address to check
     function _mustSkipFees(address from) internal virtual returns (bool);
 
+    /// @notice return the quadId given and index, size and coordinates
+    /// @param i the index to be added to x,y to get row and column
+    /// @param size The bottom left x coordinate of the quad
+    /// @param x The bottom left x coordinate of the quad
+    /// @param y The bottom left y coordinate of the quad
+    /// @return the tokenId of the quad
+    /// @dev this method is gas optimized, must be called with verified x,y and size, after a call to _isValidQuad
```

Comment:
I don't see a call to `_isValidQuad `? Should we include it?

Reply:
`_isValidQuad` is called within `batchTransferQuad`, so we don't need to add it separately.